src/vma/proto/dst_entry.cpp

```diff
@@ -780,3 +780,17 @@ int dst_entry::get_priority_by_tc_class(uint32_t pcp)
 	}
 	return VMA_DEFAULT_ENGRESS_MAP_PRIO;
 }
+
+bool dst_entry::update_ring_alloc_logic(int fd, lock_base& socket_lock, resource_allocation_key &ring_alloc_logic)
```

Comment:
I see no reason for this function.
We should modify do_ring_migration to public and do the below logic inside ring_alloc_logic_updater::update_field.

Reply:
The idea of data_updater is to avoid adding new functions to dst_entry.
Please use data updater instead of this function.